custom_random_forest.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from concurrent.futures import ProcessPoolExecutor
+from sklearn.base import BaseEstimator
+from sklearn.tree import DecisionTreeClassifier
+from joblib import dump, load
+import os
+import multiprocessing
```

Comment:
порядок импортов неправильный
```suggestion
import multiprocessing
import os
from concurrent.futures import ProcessPoolExecutor

import numpy as np
from joblib import dump, load
from sklearn.base import BaseEstimator
from sklearn.tree import DecisionTreeClassifier
```